src/4WA9.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Literal scalars
+  from: '@ingydotnet'
+  tags: literal
+  yaml: |
+    - aaa: |2
+        xxx
+      bbb: |
+        xxx
```

Comment:
Can we use a bit more descriptive content where possible? Avoid `aaa`, `xxx`, `foo`, `bar`?

Reply:
I'd rather do this in a separate PR.

I can see benefits and drawbacks to using descriptive data.